docs/docs/docs/getting-started/installation.md

```
@@ -291,29 +291,29 @@ These steps are specific to Linux. You will need to modify them accordingly for
-devcontainer build --workspace-folder .
-```
+   ```
+   devcontainer build --workspace-folder .
+   ```
 
 1. When the build is complete, the last line of the output should be:
 
-```
-{"outcome":"success","imageName":"talawa-api"}
-```
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)